### PR TITLE
Added the basic Navigation to navigate between the four pages

### PR DESCRIPTION
**Pull request by Alexander-Schulte**
Hey guys! Just implemented my navbar for my app. I created my four "pages" so I can navigate between them. In my Navigation.js I implemented the link to the different pages when the user is clicking on one of the specific icons. The icons are svg from my adobe xd file! Thank u for your review! :)

### CODE REVIEW

File: src/components/Navigation.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  align-items: center;
+  justify-content: space-evenly;
+  margin-top: 20px;
+}
+
+.LinkAnker {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```
Comment (FriederikeM) [inline src/components/Navigation.css:12]: What does this refer to? I don't see anything with a class "LinkAnker" in your Navigation.js

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,28 @@
 .App {
-  text-align: center;
+  display: grid;
+  height: 100vh;
+  min-width: 320px;
+  max-width: 10vw;
+  grid-template-rows: [header] 80px [main] auto [footer] 80px;
```
Comment (soerendolling) [inline src/App.css:6]: Looks like you are going from outside to inside with sorting. You might wann put the template-rows somewhere on the top as well

File: src/images/MapIcon.svg [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<svg id="MapIcon" data-name="Gruppe 44" xmlns="http://www.w3.org/2000/svg" width="39.794" height="34.351" viewBox="0 0 39.794 34.351">
```
Comment (FriederikeM) [inline src/images/MapIcon.svg:1]: I like that you are using your own SVGs, very cool! 😎

**Review by FriederikeM — APPROVED** [review]
Looks mostly good, just two minor comments

**Review by soerendolling — APPROVED** [review]
Just noticed something small. Everything else looks fine to me.

File: src/App.js [hunk 4]
```diff
@@ -1,24 +1,39 @@
...
-      </header>
-    </div>
+    <Router>
+      <div className="App">
+        <header className="Header">
+          <p>This is the header!</p>
+        </header>
+        <main className="Main">
+          <Switch>
+            <Route exact path="/">
```
Comment (hugh-burgess) [inline src/App.js:18]: if im not mistaken, i think the home one goes last, but check!

**Review by hugh-burgess — CHANGES_REQUESTED** [review]
Looks To Me Good!

File: src/App.js (hunk 4 above)
Comment (namirsab) [reply src/App.js:18]: Setting the order of the routes from more specific to more generic it's a good practice.
https://reactrouter.com/web/guides/primary-components/route-matchers

File: src/App.css [hunk 5]
```diff
@@ -1,38 +1,28 @@
...
+  font-size: 16px;
 }
 
-.App-link {
-  color: #61dafb;
+.Main {
+  display: flex;
+  grid-row: main;
+  justify-content: center;
+  width: 100vw;
```
Comment (namirsab) [inline src/App.css:23]: I'm not sure what you are trying to achieve, but I see that you've set the `max-width` in the app to `10vw` but here you are setting now the `Main` with to `100vw`.

**Review by namirsab — APPROVED** [review]
Very good Alex!
Just a few comments, but nothing critical

**Pull request merged**